great3sims/psf.py

```diff
@@ -149,7 +149,7 @@ class ConstPSFBuilder(PSFBuilder):
     # For lam/diam, Euclid has ~0.1375".
     #               WFIRST has a range for the different bands, but with 2.4m mirror and 1.1 micron,
     #               it is 0.0945".
-    # In terms of sampling, we use the correspondence between OpticalPSF <-> Airy 
```

Comment:
Throughout, there are a lot of trivial whitespace changes. I agree that they are an improvement, but it's not appropriate to include them on the same commit as "real" work.